manager/manager.go

```diff
@@ -783,6 +790,35 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// Enables overwriting an existing containerData/Handler object for a given containerName.
+// can't use createContainer as it just returns if a given containerName has a handler already
+// Ex: rkt handler will want to take priority over the raw handler, but the raw handler might be created first
+
+// Only allow raw handler to be overriden
+func (m *manager) overrideContainer(containerName string, watchSource watcher.ContainerWatchSource) error {
+	m.containersLock.Lock()
+	defer m.containersLock.Unlock()
+
+	namespacedName := namespacedContainerName{
+		Name: containerName,
+	}
+
+	if _, ok := m.containers[namespacedName]; ok {
+		containerData := m.containers[namespacedName]
+
+		if containerData.handler.String() != "raw" {
+			return nil
+		}
```

Comment:
don't want to waste time destroying a rkt handler and replacing it with a rkt handler.

i.e. cadvisor restarts so rkt container handler will work for cgroups that exist, a little later the rkt watcher sends those same cgroups, no need to destroy/recreate